### PR TITLE
Unify table reported when `thamos check' called

### PR DESCRIPTION
**Pull request by fridex**
## This introduces a breaking change

- [x] No

## Description

Let's unify how `thamos check` reports results with other tables reported.

Related: #988

### CODE REVIEW

File: thamos/cli.py [hunk 1]
```diff
@@ -1086,36 +1086,11 @@ def check(runtime_environment: Optional[str], output_format: str) -> None:  # no
...
-                row.append(entry if entry is not None else "-")
-
-            table.add_row(*row)
-
-        console = Console()
-        console.print(table, justify="center")
+        _print_report(
+            result,
+            json_output=False,
+            title="Runtime environment and cofniguration check results",
```
Comment (mayaCostantini) [inline thamos/cli.py:1092]:
```suggestion
            title="Runtime environment and configuration check results",
```

**Review by mayaCostantini — COMMENTED** [review]
One typo but otherwise lgtm :100:

**Comment by fridex** [comment]
> One typo but otherwise lgtm 100

Adjusted 😅 Thanks for spotting it 👍🏻

**Comment by mayaCostantini** [comment]
> Adjusted sweat_smile Thanks for spotting it 👍🏻

No problem :100: 

/approve

**Pull request merged**